webknossos/webknossos/dataset/defaults.py

```diff
@@ -1,13 +1,21 @@
+import os
+
 from ..geometry import Vec3Int
 from .data_format import DataFormat
 
 DEFAULT_WKW_FILE_LEN = 32
 DEFAULT_CHUNK_SHAPE = Vec3Int.full(32)
-DEFAULT_CHUNKS_PER_SHARD = Vec3Int.full(32)
+DEFAULT_DATA_FORMAT = (
+    DataFormat.WKW
+    if not os.environ.get("WK_DEFAULT_DATA_FORMAT") == "zarr3"
+    else DataFormat.Zarr3
+)
+DEFAULT_CHUNKS_PER_SHARD = Vec3Int.full(
```

Comment:
Ideally, the env var could be a scalar for `Vec3Int.full()` or a comma-separated 3-tuple for `Vec3Int()`.